core/src/test/resources/application.conf

```diff
@@ -52,8 +52,8 @@ database-test-mariadb {
   # mysql --protocol=tcp -P13306 -ucromwell -ptest -e "DROP DATABASE IF EXISTS cromwell_test; CREATE DATABASE cromwell_test;"
   profile = "slick.jdbc.MySQLProfile$"
   db {
-    driver = "com.mysql.cj.jdbc.Driver"
-    url = "jdbc:mysql://localhost:13306/cromwell_test?useSSL=false&rewriteBatchedStatements=true&serverTimezone=UTC"
+    driver = "org.mariadb.jdbc.Driver"
+    url = "jdbc:mariadb://localhost:13306/cromwell_test?rewriteBatchedStatements=true"
```

Comment:
Out of curiosity, what was the effect till now when we were connecting to mysql driver instead of mariadb?

Reply:
Think of it like using Firefox to connect to https://google.com. Or Chrome to connect to http://mozilla.org. It mostly works, but if one filed a bug for the client talking to the server I wouldn't expect the quickest of fixes.

The MySQL JDBC driver works "fine" talking to a MariaDB server. We did discover at least one issue though: https://broadworkbench.atlassian.net/browse/BA-5692